Files.sln

```diff
@@ -64,216 +68,288 @@ Global
 		Store|x86 = Store|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{94F77692-D47C-48D8-A1A7-645192EF38A4}.Debug|Any CPU.ActiveCfg = Debug|x64
```

Comment:
Still out there.
You can remove the configuration from Configuration Manager?

Reply:
Every addition regarding Any CPU should be removed

Reply:
I see you have committed changes, is this now resolved?

Reply:
Yes